client_apis/python/src/tests/integration/alerts_test.py

```diff
@@ -0,0 +1,212 @@
+        self.assertNotEqual(len(alerts['results']), 0)
+        alert = alerts['results'][0]
+
+        token_header = {'X-Auth-Token': token}
+
+        r = requests.post("%s/api/v1/alert/%s" % (url, alert['unique_id'] + '123'), headers=token_header,
+                          data=json.dumps(alert), verify=False)
+        self.assertEqual(r.status_code, 500)
+
+        r = requests.post("%s/api/v1/alert/%s" % (url, alert['unique_id'] + '123'), headers=token_header,
+                          data=json.dumps(alert), verify=False)
+        self.assertEqual(r.status_code, 500)
+
+        # GET
+        r = requests.get("%s/api/v1/alert/%s" % (url, alert['unique_id']), headers=token_header, verify=False)
+        self.assertEqual(r.status_code, 405)
+        # DELETE
+        r = requests.delete("%s/api/v1/alert/%s" % (url, alert['unique_id']), headers=token_header, verify=False)
+        self.assertEqual(r.status_code, 405)
+        # PUT
+        r = requests.put("%s/api/v1/alert/%s" % (url, alert['unique_id']), headers=token_header, verify=False)
+        self.assertEqual(r.status_code, 405)
+
+        # /api/v1/alert
+
+        # DELETE
+        r = requests.delete("%s/api/v1/alert" % url, headers=token_header, verify=False)
+        self.assertEqual(r.status_code, 405)
+        # PUT
+        r = requests.put("%s/api/v1/alert" % url, headers=token_header, verify=False)
+        self.assertEqual(r.status_code, 405)
+
+        # /api/v1/alerts
+
+        # GET
+        r = requests.get("%s/api/v1/alerts" % url, headers=token_header, verify=False)
+        self.assertEqual(r.status_code, 405)
+        # DELETE
+        r = requests.delete("%s/api/v1/alerts" % url, headers=token_header, verify=False)
+        self.assertEqual(r.status_code, 405)
+        # PUT
+        r = requests.put("%s/api/v1/alerts" % url, headers=token_header, verify=False)
+        self.assertEqual(r.status_code, 405)
+
+        # NON-AUTH
+        # /api/v1/alert
+        r = requests.get("%s/api/v1/alert" % url, headers='', verify=False)
+        self.assertEqual(r.status_code, 403)
+        # /api/v1/alert/id
+        r = requests.post("%s/api/v1/alert/%s" % (url,'1234'), headers='', verify=False)
+        self.assertEqual(r.status_code, 403)
+        # /api/v1/alerts
+        r = requests.post("%s/api/v1/alerts" % url, headers='', verify=False)
+        self.assertEqual(r.status_code, 403)
+
+    @classmethod
+    def tearDownClass(cls):
+        cb.watchlist_del(cls.watchlist_id)
+        return
+
+
+
+if __name__ == '__main__':
+    if 5 != len(sys.argv):
+        print "usage   : python alerts_test.py server api_token sshuser sshuser_password"
+        print "example : python alerts_test.py cb.my.org 3ab23b1bdhjj3jdjcjhh2kl user password\n"
+        sys.exit(0)
+
+    # instantiate a global CbApi object
+    # all unit tests will use this object
+    #
+    server = sys.argv[1]
+    token = sys.argv[2]
+    user = sys.argv[3]
```

Comment:
I'd include "ssh" in the name of these params to make it clear that it's not about credentials for the cb server app itself